conv/abci/block_test.go

```diff
@@ -0,0 +1,235 @@
+package abci
+
+import (
+	cmbytes "github.com/cometbft/cometbft/libs/bytes"
+
+	cmproto "github.com/cometbft/cometbft/proto/tendermint/types"
+
+	"testing"
+	"time"
+
+	cmversion "github.com/cometbft/cometbft/proto/tendermint/version"
+	cmtypes "github.com/cometbft/cometbft/types"
+	"github.com/rollkit/rollkit/types"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestToABCIHeaderPB(t *testing.T) {
+	header := &types.Header{
```

Comment:
Let's pull this into a helper method so that we can re-use it in the following tests